alacritty_terminal/src/term/mod.rs

```diff
@@ -1234,11 +1236,12 @@ impl<T> Term<T> {
     where
         T: EventListener,
     {
-        // Setting 132 column font makes no sense, but run the other side effects
-        // Clear scrolling region
+        // Setting 132 column font makes no sense, but run the other side effects.
+        //
+        // Clear scrolling region.
```

Comment:
Why did you change the comment style to this? The newline seems counter-productive for simple comments like this.